docs/content/routing/services/index.md

```diff
@@ -311,14 +311,15 @@ On subsequent requests, to keep the session alive with the same server, the clie
 #### Health Check
 
 Configure health check to remove unhealthy servers from the load balancing rotation.
-Traefik will consider your servers healthy as long as they return status codes between `2XX` and `3XX` to the health check requests (carried out every `interval`).
+Traefik will consider your HTTP(s) servers healthy as long as they return status codes between `2XX` and `3XX` to the health check requests (carried out every `interval`).
+For gRPC servers, Traefik will consider them healthy as long as they return `SERVING` to [gRPC health check v1](https://github.com/grpc/grpc/blob/master/doc/health-checking.md) requests. You have to define the `scheme` to `grpc`.
 
 To propagate status changes (e.g. all servers of this service are down) upwards, HealthCheck must also be enabled on the parent(s) of this service.
 
 Below are the available options for the health check mechanism:
 
 - `path` is appended to the server URL to set the health check endpoint.
-- `scheme`, if defined, will replace the server URL `scheme` for the health check endpoint
+- `scheme`, if defined to `grpc`, will use the gRPC health check protocol to probe the server. If defined to something else, will replace the server URL `scheme` for the health check endpoint.
```

Comment:
Could you create another field `mode` with default value "http" and a possible value "grpc"?
`scheme` is used if you want to use "https" or "http" and it still a possible choice even with grpc healthcheck.

Reply:
Done, i added the `mode` option to healthCheck and updated doc and tests accordingly.